models/cdm/cdm/Fast.py

```diff
@@ -49,10 +49,8 @@ def initialize(self, model, **kwds):
         libcdm.oid(source, oid)
         # inform the source about the parameter layout; assumes contiguous parameter sets
         libcdm.layout(source,
-                      model.xIdx, model.dIdx,
-                      model.openingIdx, model.aXIdx, model.omegaXIdx,
+                      model.xIdx, model.dIdx, model.openingIdx, model.aXIdx, model.omegaXIdx,
```

Comment:
Modified function variable referencing/order for consistency.

Reply:
this looks like merging two lines into one; am i missing something